<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliderDemoWidget</class>
 <widget class="QWidget" name="SliderDemoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>356</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="animateProgressBarCheckBox">
     <property name="text">
      <string>Animate Progress Bars</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="invertProgressBarCheckBox">
     <property name="text">
      <string>Invert Progress Bars</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Horizontal</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="horizontalProgressBar">
        <property name="value">
         <number>25</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="horizontalScrollBar">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dial">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="notchesVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Vertical</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSlider" name="verticalSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="verticalProgressBar">
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sliders tick marks position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tickPositionComboBox">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>No Tick Marks</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tick Marks Above Slider</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tick Marks Below Slider</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tick Marks On Both Sides</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>animateProgressBarCheckBox</tabstop>
  <tabstop>invertProgressBarCheckBox</tabstop>
  <tabstop>tickPositionComboBox</tabstop>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>dial</tabstop>
  <tabstop>verticalSlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
